src/core/thread/fiber.d

```diff
@@ -958,6 +958,13 @@ private:
         {
             version (Posix) import core.sys.posix.sys.mman; // mmap, MAP_ANON
 
+            static if ( __traits( compiles, ucontext_t ) )
+            {
+                // Stack size must be at least the minimum allowable by the OS.
+                if (sz < MINSIGSTKSZ)
+                    sz = MINSIGSTKSZ;
```

Comment:
I may come back to this if Darwin still craps out with a smaller stack using the asm Fibers.

Reply:
On a related note, @Geod24 are you sorting out the fiber stack overflow test you have a PR for? The minimum stack size should be at least this to trigger overflow on ucontext targets.

Reply:
On a related note, @Geod24 are you sorting out the fiber stack overflow test you have a PR for? The minimum stack size should be at least this to trigger overflow on ucontext targets.